templates/taxi/manufacturer_list.html

```diff
@@ -23,6 +26,12 @@ <h1>Manufacturer List
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                 <a class="btn btn-primary" href="{% url 'taxi:manufacturer-update' manufacturer.id %}">Update</a>
```

Comment:
The URL tag uses `manufacturer.id`, but it should be `manufacturer.pk` to ensure consistency with Django's convention for primary keys. Consider changing `manufacturer.id` to `manufacturer.pk`.